packages/fake-driver/test/general-tests.js

```diff
@@ -1,7 +1,7 @@
 import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
-import { initSession, deleteSession, DEFAULT_CAPS } from './helpers';
-import { W3CActions } from 'wd';
+import chaiWebdriverIOAsync from 'chai-webdriverio-async';
```

Comment:
o_O there's a package for this??

Reply:
yeah.  async webdriver isn't the most fun imo... this makes it a little easier. sometimes.